contracts/oracle/CollateralizationOracleWrapper.sol

```diff
@@ -110,13 +112,28 @@ contract CollateralizationOracleWrapper is Timed, ICollateralizationOracle, Core
     /// @notice set the validity duration of the cached collateralization values.
     /// @param _validityDuration the new validity duration
     /// This function will emit a DurationUpdate event from Timed.sol
-    function setValidityDuration(uint256 _validityDuration) external onlyGovernor {
+    function setValidityDuration(uint256 _validityDuration) external override onlyGovernor {
         _setDuration(_validityDuration);
     }
 
     // ----------- IOracle override methods ------------------------------------
     /// @notice update reading of the CollateralizationOracle
     function update() external override whenNotPaused {
+        _update();
+    }
+
+    /** 
+        @notice this method reverts if the oracle is not outdated
+        It is useful if the caller is incentivized for calling only when the deviation threshold or frequency has passed
+    */ 
+    function updateIfOutdated() external override whenNotPaused {
+        require(_update(), "CollateralizationOracleWrapper: outdated");
```

Comment:
the exception message would rather be "not outdated"

Reply:
Fixed